contracts/mainnet/connectors/notional/events.sol

```diff
@@ -2,90 +2,118 @@
 pragma solidity ^0.7.6;
 
 contract Events {
-    event LogDepositCollateral(
```

Comment:
@weitianjie2000 can we revert some of the formatting errors

Reply:
done